starknet/l2_minter/src/interfaces/l1_minter.cairo

```diff
@@ -0,0 +1,17 @@
+use starknet::class_hash::ClassHash;
+
+#[starknet::interface]
+trait IEthereumMinter<TContractState> {
+    fn get_l1_minter_address(self: @TContractState) -> felt252;
+    fn set_l1_minter_address(ref self: TContractState, l1_address: felt252);
+    // #[l1_handler]
+    // fn mint_value(
```

Comment:
It cannot be defined yet (due to cairo current version)?

Reply:
Yes, I'm not sure how those are supposed to be defined.
Right now it doesn"t matter as I'm not using the interface anymore, it is derived using #[generate_trait]